libs/wingsdk/src/target-sim/table.inflight.ts

```diff
@@ -13,16 +13,26 @@ export class Table implements ITableClient, ISimulatorResourceInstance {
   private primaryKey: string;
   private table: Map<string, any>;
   private readonly context: ISimulatorContext;
+  private readonly initialRows: Record<string, Json>;
 
   public constructor(props: TableSchema["props"], context: ISimulatorContext) {
     this.name = props.name;
     this.columns = props.columns;
     this.primaryKey = props.primaryKey;
     this.table = new Map<string, any>();
     this.context = context;
+    this.initialRows = props.initialRows ?? {};
   }
 
   public async init(): Promise<TableAttributes> {
+    for (const [key, row] of Object.entries(this.initialRows)) {
+      await this.context.withTrace({
+        message: `Adding row from preflight (key=${key}).`,
```

Comment:
I think logging from `preflight` might be a little too technical. Maybe something like this
```suggestion
        message: `Adding initial row (key=${key}).`,
```